USF-1670: Dropins Localization (#95)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/user-auth/auth-dictionary.mdx

---

> title: User Authentication Dictionary

Reviewer: ```suggestion
title: User authentication dictionary
```

> description: Reference the default dictionary for the User Authentication dropin.

Reviewer: ```suggestion
description: Learn how to customize the default dictionary for the user authentication drop-in component.
```

---

import CodeImport from '@components/CodeImport.astro';
import CodeInclude from '@components/CodeInclude.astro';
import dictionary from '@dropins/storefront-auth/i18n/en_US.json.d.ts?raw';

> The default User Authentication dictionary file (`i18n/en_US.json`) contains the default values for all labels and text common to the this drop-in.

Reviewer: ```suggestion
The default user authentication dictionary file (`i18n/en_US.json`) contains the default values for all labels and text common to the user authentication drop-in component.
```
